extensions/mongoProcessor/utils/contentTypeHelper.js

```diff
@@ -0,0 +1,83 @@
+'use strict'; // eslint-disable-line
+
+function _getDataContent(entry) {
+    const contentLength = entry.getContentLength();
+    if (contentLength > 0) {
+        return ['DATA', 'METADATA'];
```

Comment:
Is it possible that we also need to do a `PUT_TAGGING` in case the kafka entry contains tags? Maybe it would not happen or it would always be a separate event, just wondering.

Reply:
ty updated, this definitely applies for new objects with tagging

Reply:
Seems like new objects w/ tagging will not replicate the tagging with it. Created a ticket (1707) for tracking